std_msgs/String secret_code_front
---
std_msgs/String secret_code_next